website/docs/r/team.html.markdown

```diff
@@ -45,10 +41,10 @@ In addition to all arguments above, the following attributes are exported:
 
 ## Import
 
-Clusters can be imported using the organization ID and team id, in the format `ORGID-TEAMID-PROJECTID`, e.g.
+Clusters can be imported using the organization ID and team id, in the format `ORGID-TEAMID`, e.g.
```

Comment:
One minor change! 
Teams can be imported using the organization ID and team id, in the format `ORGID-TEAMID`, e.g.